logging/config.js.example

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
The config file itself isn't versioned so you can have different configs for different deploys.